README.md

```diff
@@ -32,68 +79,144 @@ and [test](https://en.wikipedia.org/wiki/Training,_validation,_and_test_sets#Tes
     - availables parameters: `t2m(2t)/siconc(ci)/gh/lsm/msl/q/rsn/sp/sst/t/tcc/tcw/tp/ttr/u/v` (differing name in [MARS database](https://confluence.ecmwf.int/display/S2S/Parameters))
-    - parameters not available: sm20, sm100, st20, st100
+    - parameters not available: `sm20,sm100,st20,st100`
   - NCEP hindcast data 
-    - `forecast_time` : from 1999/01/07 to 2010/12/30, weekly every 7 days (every Thurday).
+    - `forecast_time` : from 1999/01/07 to 2010/12/30, corresponding to the weekly Thurdays in 2010.
     - `lead_time` : 1 to 44 days
     - `valid_time` (`forecast_time` + `lead_time`): from 1999/01/07 to 2011/02/11
     - availables parameters: `t2m(2t)/siconc(ci)/gh/lsm/msl/q/sm100/sm20/sp/sst/st100/st20/t/tcc/tcw/tp/ttr/u/v` (differing name in [MARS database](https://confluence.ecmwf.int/display/S2S/Parameters))
     - parameter not available: `rsn`
 
+```python
+hindcast = climetlab.load_dataset('s2s-ai-challenge-training-input', date=[20200102], origin='ecwmf', parameter='tp', format='netcdf').to_xarray()
+hindcast.coords
+Coordinates:
+  * realization    (realization) int64 0 1 2 3 4 5 6 7 8 9 10
+  * forecast_time  (forecast_time) datetime64[ns] 2000-01-02 ... 2019-01-02
+  * lead_time      (lead_time) timedelta64[ns] 0 days 1 days ... 45 days 46 days
+  * latitude       (latitude) float64 90.0 88.5 87.0 85.5 ... -87.0 -88.5 -90.0
+  * longitude      (longitude) float64 0.0 1.5 3.0 4.5 ... 355.5 357.0 358.5
+    valid_time     (forecast_time, lead_time) datetime64[ns] 2000-01-02 ... 2...
+    
+# for ncep hindcast provide 2010 date strings
+hindcast_ncep = climetlab.load_dataset('s2s-ai-challenge-training-input', date=[20100107], origin='ncep', parameter='tp', format='netcdf').to_xarray()
+hindcast_ncep.coords
+Coordinates:
+  * realization    (realization) int64 0 1 2 3
+  * forecast_time  (forecast_time) datetime64[ns] 1999-01-07 ... 2010-01-07
+  * lead_time      (lead_time) timedelta64[ns] 1 days 2 days ... 43 days 44 days
+  * latitude       (latitude) float64 90.0 88.5 87.0 85.5 ... -87.0 -88.5 -90.0
+  * longitude      (longitude) float64 0.0 1.5 3.0 4.5 ... 355.5 357.0 358.5
+    valid_time     (forecast_time, lead_time) datetime64[ns] 1999-01-08 ... 2...
+```
+
  List of files :
   [grib](https://storage.ecmwf.europeanweather.cloud/s2s-ai-challenge/data/training-input/0.3.0/grib/index.html),
   [netcdf](https://storage.ecmwf.europeanweather.cloud/s2s-ai-challenge/data/training-input/0.3.0/netcdf/index.html),
   zarr (not available)
 
-### Forecast input (Test input)
+
+### Forecast input
+
 The `forecast-input` (`test-input`) dataset consists also in data from three different models: ECMWF (ecmf), ECCC (cwao), NCEP (eccc), for different dates.
 These data are forecast data.
 This could be used the input for applying the ML models in order to generate the output which is submitted for the challenge.
-Using data from earlier date that 2020/01/01 is also allowed during the prediction phase.
```

Comment:
Is is documented elsewhere ?
If it is not allowed, maybe add "Using data from earlier date that 2020/01/01 is *not* allowed during the prediction phase."

Reply:
But should it be forbidden?

Reply:
IMO it is allowed to use a 3month prediction issued in October 2019 to predict Jan 2020

Reply:
I agree, that's why I would keep the text "Using data from earlier date that 2020/01/01 is also allowed during the prediction phase". I don't understand why you removed it, except if it is documented elsewhere.

Reply:
I removed it because I think it is unnecessarily. furthermore, the rules are defined on the website. climetlab is just to get the data IMO.